src/main/kotlin/org/openrewrite/kotlin/KotlinTypeMapping.kt

```diff
@@ -722,14 +722,18 @@ class KotlinTypeMapping(typeCache: JavaTypeCache, firSession: FirSession) : Java
             simpleFunction = symbol.fir as FirSimpleFunction
         }
         var paramNames: MutableList<String>? = null
+        if (simpleFunction?.receiverParameter != null) {
+            paramNames = ArrayList(simpleFunction.valueParameters.size + 1)
+            paramNames.add("receiver")
```

Comment:
I need to check what the actual name of this parameter is. Does Kotlin generate it into the bytecode? Is it fixed?

Reply:
Fastest way to answer this question is probably to hit a breakpoint inside an extension function and see what intelliJ calls it

Reply:
I decompiled a class file and it looks like the parameter name is `$this$<extension-name>`. I left it at `$this$` for now.